.changes/next-release/Feature-ae9e0065-3fe4-4f9f-a812-409097f2ebe9.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Add enforceDockerNamingConventions to ECRPushImage"
```

Comment:
nitpick: Seems reasonable to give a bit more detail

Reply:
Agreed. Are there other completed feature contributions that I can compare to? I wasn't quite sure what level of detail needed to be included here.

Reply:
Typing it out it actually seems redundant, so this is fine as is (with a rename to match the parameter name)

Reply:
Oops, missed your comment. I just pushed an update. Hope that's ok?